src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRunCodegenNodeFactory.cs

```diff
@@ -443,6 +448,12 @@ public override void AttachToDependencyGraph(DependencyAnalyzerBase<NodeFactory>
             RuntimeFunctionsGCInfo = new RuntimeFunctionsGCInfoNode();
             graph.AddRoot(RuntimeFunctionsGCInfo, "GC info is always generated");
 
+            if (_ibcTuning)
```

Comment:
Do we need this bool? Would things not naturally fall out? (No profile data means ProfileDataSection reports ShouldSkipEmitting = true, which means ready to run header won't create a reference to it.)

Just looking if we can remove complexity.